frontend/src/metabase/visualizations/visualizations/LineChart/LineChart.stories.tsx

```diff
@@ -67,3 +67,10 @@ export const EmbeddingHugeFont: Story = () => {
     </SdkVisualizationWrapper>
   );
 };
+
+// This story has become flaky on CI, so we're skipping it for now.
+EmbeddingHugeFont.story = {
```

Comment:
It's for the SDK's themed variations that is flaking - the default-theme variation is fine. Let's skip it for now to unblock CI, and investigate it later.